source/_docs/scripts.markdown

```diff
@@ -657,7 +657,10 @@ automation:
 
 {% endraw %}
 
-`conditions` also accepts a [shorthand notation of a template condition][shorthand-template].
+`conditions` also accepts a [shorthand notation of a template condition][shorthand-template]. <br>
+It needs to be wrapped in quotation marks (```""```) unless using folding (```>```) where it must not use quotation marks. When wrapping in outside double quotation marks ensure any inner quotation marks are of the single (```'```) variant. <br>
```

Comment:
- This is actually partially incorrect and not related to scripts at all, this is generic YAML stuff, which doesn't belong here.
- Do not use HTML in Markdown

Reply:
Thanks for all the detailed respones!
 
We can argue that this is not a yaml tutorial site, but I spent quite some time figuring it all out. It does not come at a cost to include some info here. A generic page with all ins and outs of yaml (HA specific) with links from all doc pages would be very handy as well. Not all of us are yaml proficient.

The fact that, even after research, I seem to be incorrect (can you please state what is incorrect) shows that it needs explanation. Sometimes list, sometimes fold, sometimes single apostrophes, sometimes double apostrophes. Please look through the eyes of a new HA/yaml user.

Reply:
> We can argue that this is not a yaml tutorial site

We don't have to argue that. We are simply not.